### PR TITLE
add rollback rpc definition

### PR DESCRIPTION
**Pull request by ianedwards**

### CODE REVIEW

File: porter/v1/cluster_control_plane.proto [hunk 1]
```diff
@@ -240,6 +240,23 @@ message UpdateRevisionStatusRequest {
 
 message UpdateRevisionStatusResponse {}
 
+message RollbackRevisionRequest {
+  int64 project_id = 1;
```
Comment (ferozemohideen) [inline porter/v1/cluster_control_plane.proto:244]: comment for project_id?

File: porter/v1/cluster_control_plane.proto [hunk 2]
```diff
@@ -240,6 +240,23 @@ message UpdateRevisionStatusRequest {
...
+  // app_id is the id of the PorterApp to roll back
+  int64 app_id = 2;
+  // deployment_target_id is the id of the deployment target where the app will be installed
```
Comment (ferozemohideen) [inline porter/v1/cluster_control_plane.proto:247]: where the app is currently installed?

**Review by ferozemohideen — APPROVED** [review]

File: porter/v1/cluster_control_plane.proto (hunk 1 above)
Comment (ianedwards) [reply porter/v1/cluster_control_plane.proto:244]: we haven't been adding a comment for project_id so just maintaining consistency I guess

File: porter/v1/cluster_control_plane.proto (hunk 2 above)
Comment (ianedwards) [reply porter/v1/cluster_control_plane.proto]: updated

**Pull request merged**